lib/schemas/config.js

```diff
@@ -23,6 +23,7 @@ const configSchema = yup.object().shape({
     .positive()
     .required(),
   verbose: yup.boolean().required(),
+  quiet: yup.boolean().required(),
   supportedLocales: yup.array().nullable(),
```

Comment:
Given this validation assumes the default configs, can we also add a test or 2 in `test/config.test.js` against a `minimalConfig`?

Reply:
I think the only props that are strictly necessary are `clientId` and `clientSecret`